pkg/azurefileplugin/Dockerfile

```diff
@@ -21,6 +21,11 @@ COPY ${binary} /azurefileplugin
 RUN apk upgrade --available --no-cache && \
     apk add --no-cache ca-certificates cifs-utils util-linux e2fsprogs-extra udev xfsprogs-extra nfs-utils
 
+RUN apk add gcompat
+RUN wget -c https://azcopyvnext.azureedge.net/release20230420/azcopy_linux_amd64_10.18.1.tar.gz && tar -zxvf azcopy_linux_amd64_10.18.1.tar.gz && \
```

Comment:
is there an arm64 support for this binary?

Reply:
I think it is not supported because `azcopy` has another arm64 preview binary (https://aka.ms/downloadazcopy-v10-linux-arm64).

Reply:
then copy arm64 binary if it's arm64 image